### PR TITLE
Enable incremental linking for ImageProcessingAtom debug builds.

### PR DESCRIPTION
**Pull request by hershey5045**
Tested by running cmake for Windows vs2019 and verifying that the "Enable Incremental Linking" option in the ImageProcessingAtom.Editor project is enabled. 

Signed-off-by: hershey5045 <43485729+hershey5045@users.noreply.github.com>

### CODE REVIEW

**Review by galibzon — APPROVED** [review]

**Review by gadams3 — APPROVED** [review]

**Review by santorac — APPROVED** [review]

**Comment by lawsonamzn** [comment]
do we have anyone who has a repro case of this and can verify it works?

File: Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Platform/Windows/platform_windows.cmake [hunk 1]
```diff
@@ -6,3 +6,5 @@
 #
 #
 
+# Temporary fix for the linker issue caused by OpenEXR libs on debug builds
+set(LY_LINK_OPTIONS INTERFACE "$<$<CONFIG:debug>:/INCREMENTAL>")
```
Comment (lumberyard-employee-dm) [inline Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Platform/Windows/platform_windows.cmake:10]: This linkere option should be placed in the compiler specific cmake settings file located in the [Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Platform/Common](https://github.com/o3de/o3de/tree/abb635e583436d72d5e9a9b712dbb7b448cc94e6/Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Platform/Common) directory.

**Review by lumberyard-employee-dm — CHANGES_REQUESTED** [review]

File: Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Platform/Windows/platform_windows.cmake (hunk 1 above)
Comment (hershey5045) [reply Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Platform/Windows/platform_windows.cmake:10]: oh awesome that would make more sense.

**Comment by hershey5045** [comment]
Workaround no longer needed, solution was to update visual studio to the latest.

**Pull request closed without merging**